packages/fx-core/src/common/constants.ts

```diff
@@ -75,6 +75,7 @@ export class FeatureFlagName {
   static readonly YoCheckerEnable = "TEAMSFX_YO_ENV_CHECKER_ENABLE";
   static readonly GeneratorCheckerEnable = "TEAMSFX_GENERATOR_ENV_CHECKER_ENABLE";
   static readonly ApiConnect = "TEAMSFX_API_CONNECT_ENABLE";
+  static readonly DeployManifest = "TEAMSFX_DEPLOY_MANIFEST";
```

Comment:
Could you please help to add this to `initializeGAFeatureFlags()` as well?

Reply:
Sure, added. Thx for reminding.